### ISSUE TITLE
Update all language files for 0.12.1 - Call for translators

### ISSUE THREAD

**Issue by erciccione** [monero-project/monero-gui#1377]
The GUI is frozen, now we need to update all existent languages. Keep in mind that the new GUI will be released once all languages will be updated, so please, if you are working on something else, **give the maximum priority to the GUI**.

 ~~I refreshed all language files (see #1376 ), you can find the uploaded version of each language on the `refresh-lang` branch of my repo. Please **use these updated files, NOT the ones on the official repository**. Updated language files can be found at this link:~~ 

~~https://github.com/erciccione/monero-core/tree/refresh-lang/translations~~

Pull the file related to your language from ~~that link~~

https://github.com/monero-project/monero-gui/tree/master/translations

translate the missing strings and, when done, open a PR against monero-project/monero-gui referencing to this issue.

**! Do not update the .ts file and don't run any command, just pull the correct file for your language and translate it. Stick to this workflow, please. !**

## Languages and tasks
Refer to the Taiga task to know the status of each language (if a task's status is 'in progress' or 'assigned to', somebody else is already working on it). I listed down here all available languages, as soon as a translation is PRd, the correspondent language will be thicked.

- [ ] **Arabic** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/255)
- [ ] **Catalan** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/256)
- [x] *Merged* **Czech** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/284)
- [x] *merged* **Danish** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/260)
- [x] *Merged* **German** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/264)
- [x] *Merged* **Esperanto** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/262)
- [x] *Merged* **Spanish** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/277)
- [x] *merged* **French** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/263)
- [x] *Merged* **Hebrew** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/265)
- [ ] **Croatian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/259)
- [ ] **Indonesian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/266)
- [x] *Merged* **Italian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/267)
- [x] *Merged* **Japanese** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/268)
- [ ] **Korean** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/269)
- [x] *Merged* **Dutch** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/261)
- [x] *Merged* **Polish** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/270)
- [x] *Merged* **Portuguese Brazilian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/272)
- [x] *merged* **Portuguese** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/271)
- [x] *Merged* **Romanian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/273)
- [x] *Merged* **Russian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/274)
- [x] *merged* **Slovak** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/283)
- [ ] **Slovenian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/276)
- [x] *Merged* **Serbian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/275)
- [x] *Merged* **Swedish** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/278)
- [x] *Merged* **Turkish** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/279)
- [ ] **Ukrainian** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/280)
- [x] *Merged* **Chinese Traditional** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/257)
- [x] *merged* **Chinese Simplified** - [Taiga task](https://taiga.getmonero.org/project/erciccione-monero-localization/task/258)

## Workflow
Please read our [General guidlines for translators](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/home) before you start.

If you are on the localization team on Taiga, use this workflow:

- I opened a new task called "Updates for release 0.12.1" and marked it as "needed" for each of the 29 languages available. Once you are actively working on the translation, change its status to 'in progress' (and preferably, assign the task to yourself).
- All needed translations will be marked as 'needed'. if somebody is already working on your language, contact the translator (here on GitHub or on Taiga) and offer him/her your help. *Do not* start translating anyway. Ask if in dubt.
- Once your work is ready and pushed on GitHub, add the link of the PR in the apposite custom field and change the status of the task to 'ready for tests'.

**If you are not on Taiga but you are working on a translation, just comment here signaling you started working on a language, and keep us updated on the status of your work. I will keep Taiga updated for you. This will avoid double works**

## Help/Support
If you need any help/support: comment this issue, come chat on `#monero-translations`(Freenode, [MatterMost](https://mattermost.getmonero.org/monero/channels/monero-translations), matrix/riot), or [PM me on reddit](https://www.reddit.com/message/compose/?to=ErCiccione)

Pinging past contributors and members of the localization workgroup on GitHub:

@3b7ameed @edwardlow @ruzaq @cryptobench @falcongoat @Rafficer @lovvskillz @Keksoj @i3visio @lh1008 @cryptochangements34 @glv2 @assylias @milargos @jonahar @TasmaniaKrama @hrumag @kenshi84 @takuto-h @isaacdigs @ProkhorZ @ElectricSlacks @fridzema @einsteinsfool @szogun1987 @netrik182 @rmbb @ordtrogen @mandrill-pie @fero-sk @jernejml @xmronadaily @alexej996 @snaggen @rtonline @zhizhongzhiwai @Lafudoci @el00ruobuob

**Comment by ProkhorZ** [comment]
Thanks for freezing! I'm working on the Dutch translation.

**Comment by erciccione** [comment]
thanks @ProkhorZ , i updated the task on Taiga to 'In progress'

**Comment by netrik182** [comment]
I will start working on pt-BR today

**Comment by erciccione** [comment]
thanks @netrik182

**Comment by ordtrogen** [comment]
I've been working on the Swedish one for a while and have assigned it to myself on Taiga

**Comment by takuto-h** [comment]
I'm working on the Japanese translation.

**Comment by cryptochangements34** [comment]
I'll get started on french

**Comment by Lafudoci** [comment]
I'll translate the Traditional Chinese as always. But there is a minor mistake in the languages and tasks list (in first comment) @erciccione. The Traditional Chinese in GUI basically indicates Taiwanese Chinese. Might be a typo for Simplified Chinese? BTW, I have been in the Taiga for a while, but I still can't find how to assign myself to a task? The assign text seems not clickable.

**Comment by erciccione** [comment]
@Lafudoci I'm confused about the difference in the GUI between Taiwanese, traditional and simplified. At the moment we have `monero-core_zh-tw.ts` and `monero-core_zh-cn.ts` available in the GUI. these are Chinese Taiwanese and Traditional Chinese right? so the typo should be on Taiga, where i named Simplified Chinese the Simplified Chinese translation. Am i correct?

About Taiga. Make sure you are logged in and you should see on the [task's page](https://taiga.getmonero.org/project/erciccione-monero-localization/us/59?milestone=10) the `assign or assign to me` option. You should be able to click 'assign to me'

**Comment by ParsifalX** [comment]
Any chance of adding Greek so i can start working on it?

**Comment by erciccione** [comment]
@ParsifalX I'm afraid the GUI is already frozen, too late to add another language. But if you wish to contribute there is a lot tha could be translated, starting from getmonero.org. See: https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-monero-website

**Comment by SamsungGalaxyPlayer** [comment]
@ParsifalX you can absolutely still help with Greek, but it will most likely make it in the August/September release. In the meantime, you can help with other stuff including the website like @erciccione mentioned.

**Comment by ParsifalX** [comment]
Thanks @erciccione and @SamsungGalaxyPlayer. I'll start working on getmonero.org and i'll get back to GUI before the next point release.

**Comment by Lafudoci** [comment]
@erciccione The Traditional Chinese is officially used in Taiwan `monero-core_zh-tw`. And Simplified Chinese is officially used in China `monero-core_zh-cn` and some other country. But in some regions like Hong-Kong and Singapore, they have their local Traditional and Simplified styles. They are not in GUI yet. Or it will be like `monero-core_zh-hk`.

For taiga, I didn't find `assign or assign to me option`. Here is my screen. The `Assign` in sidebar is not clickable.
![default](https://user-images.githubusercontent.com/10460270/39774124-8d4457ea-532c-11e8-9a46-63738e0b75ab.PNG)

**Comment by faudi** [comment]
I see the italian translation is already being worked on. Anything I can do?

**Comment by faudi** [comment]
@erciccione I'd be glad to! :)

**Comment by szogun1987** [comment]
I can help someone with Polish translations (e.g. review them), but I'm not able to translate all phrases.

**Comment by einsteinsfool** [comment]
@szogun1987 Feel free to review mine.

**Comment by RatusNatus** [comment]
Looks my language is already done.

**Comment by ni311** [comment]
I'd like to start working on Romanian but I cannot assign the task to me.

**Comment by erciccione** [comment]
@RatusNatus What's your language? if it's already PRd would be nice to have it reviewed

@ni311 i will move the task to 'in progress' for you. Thanks and keep us updated please :)

**Comment by erciccione** [comment]
Quick update on the status of this issue:

7 languages are still 'needed'
21 already PRd or 'in progress'

**Comment by lh1008** [comment]
Done :) https://github.com/monero-project/monero-gui/pull/1399

**Comment by fero-sk** [comment]
Working on Slovak...

**Comment by 4broaf** [comment]
Update Japanese translations for 0.12.1 #1414

**Comment by fero-sk** [comment]
Slovak: I finished 1242 lines - attached - additional lines are too dificult for me to translate :(
Can someone else continue on this work?
[monero-core_sk.ts.txt](https://github.com/monero-project/monero-gui/files/2005707/monero-core_sk.ts.txt)

**Comment by erciccione** [comment]
0.12.1 is a matter of hours now. If you worked on a translation and haven't opened a PR yet. Now it's the time

**Comment by erciccione** [comment]
0.12.2 was out some weeks ago. Thanks everybody :)
